### ISSUE TITLE
bug(select): JAWS + Firefox not voicing options while closed

### ISSUE THREAD

**Issue by jelbourn** [angular/components#23486]
_(copy of internal Google bug report for the MDC-based select)_

### Observed behavior
when you are focused on the mat-select, and use up/down arrow keys to navigate WITHOUT pressing enter (i.e., while the select is close), no changes are voiced.

### Expected behavior
changes should be voiced

### Steps to reproduce:
move focus to mat-select, do NOT press enter
press down arrow key
This issue is reproducible on JAWS 2021.2105.53 ILM and firefox 91.0 on windows, it is not reproducible on chrome 92.0.4515.131 on the same machine

This bug is not reproducible with NVDA 2021.1

It's specifically jaws and firefox.

**Comment by crisbeto** [comment]
This could point towards an issue with `LiveAnnouncer`, because we have some logic that announces the selected option test when it is selected: https://github.com/angular/components/blob/master/src/material/select/select.ts#L684

**Comment by zarend** [comment]
I am not able to reproduce. It looks like we've already fixed this.

Environment:

 - Firefox 106.0.5 (64-bit)
 - JAWS 2020.2006.12 ILM
 - Windows 10 Enterprise Version 21H2
